hail/python/hail/docs/methods/index.rst

```diff
@@ -81,15 +81,22 @@ Methods
 
 .. rubric:: Relatedness
 
-Hail provides two methods for the inference of relatedness: PLINK-style identity
-by descent and PC-Relate. :meth:`identity_by_descent` is appropriate for
-datasets containing one homogeneous population. :meth:`pc_relate` is appropriate
-for datasets containing multiple homogeneous populations and admixture.
+Hail provides three methods for the inference of relatedness: PLINK-style
+identity by descent, KING, and PC-Relate.
```

Comment:
Maybe these method names should be links to their papers / websites?

Reply:
I added references.